src/components/side-nav/side-nav.js

```diff
@@ -24,7 +24,7 @@ const SideNav = () => {
                     <NavLink className={(navigationData) => navigationData.isActive ? styles.active : null} to="/playlists"><IconButton icon={<RiPlayList2Fill />} className={styles.icon} /></NavLink>
                 </ListItem>
                 <ListItem className={styles.nav_link}>
-                    <NavLink className={(navigationData) => navigationData.isActive ? styles.active : null} to="/watch-later"><IconButton icon={<FiClock />} className={styles.icon} /></NavLink>
+                    <NavLink className={(navigationData) => navigationData.isActive ? styles.active : null} to="/watch-later"><IconButton icon={<AiOutlineClockCircle />} className={styles.icon} /></NavLink>
```

Comment:
you can write it down in the arrow function that's why you don't have to repeat your code

Reply:
Sure will do that